framework/doc/content/source/timesteppers/TimeSequenceStepperBase.md

```diff
@@ -0,0 +1,34 @@
+# TimeSequenceStepperBase
+
+`TimeSequenceStepperBase` is a base class for time steppers that use a sequence
+time values $t_i$ to produce time step sizes.
+
+## Choosing the time step size past the final time value
+
+Suppose that $t_N$ is the maximum time value provided in the sequence. This value
+may be less than the simulation end time $t_\text{end}$ given by the
+[!param](/Executioner/Transient/end_time) parameter. If this is true and the current simulation
+time $t$ is past $t_N$, then by default, there will be a single, final time step
+that jumps to the end time:
+
+!equation
+\Delta t = t_\text{end} - t_N \,.
+
+However, in many cases this is undesirable, such as when a steady state condition
+is used to terminate a transient, in which case an arbitrarily large end time
+is specified, leading to a very large time step size. This behavior can be
+altered with [!param](/Executioner/TimeSteppers/TimeSequenceStepper/use_last_dt_after_last_t).
+If set to `true`, this uses the final time step size in the sequence instead
+for all time past $t_N$:
+
+!equation
+\Delta t = t_N - t_{N-1} \,.
+
+## Failed solves
```

Comment:
would move this paragraph first, tends to happen more than final time step issues